lib/libutee/arch/arm/user_ta_entry.c

```diff
@@ -187,10 +188,16 @@ static TEE_Result entry_invoke_command(unsigned long session_id,
 
 	__utee_to_param(params, &param_types, up);
 	ta_header_save_params(param_types, params);
-
+#ifdef CFG_TEE_BENCHMARK
+	tee_add_timestamp(params[TEE_BENCH_DEF_PARAM].memref.buffer,
```

Comment:
TEE_NUM_PARAMS is set to 5 if CFG_TEE_BENCHMARK is defined, all code that handles params rely on TEE_NUM_PARAMS value (after this [cleanup](https://github.com/OP-TEE/optee_os/pull/1154)). So actually this fifth param is processed the same way as previous four.